modular_skyrat/master_files/code/modules/mob/living/emote_popup.dm

```diff
@@ -13,7 +13,19 @@
 	animate(emote_overlay, alpha = 255, time = 5, easing = BOUNCE_EASING, pixel_y = 10)
 	addtimer(CALLBACK(src, .proc/remove_emote_popup_on_mob, emote_overlay), time)
 
+/obj/proc/flick_emote_popup_on_obj(state, time)
+	var/obj/effect/overlay/emote_popup/emote_overlay = new
+	emote_overlay.icon_state = state
+	vis_contents += emote_overlay
+	animate(emote_overlay, alpha = 255, time = 5, easing = BOUNCE_EASING, pixel_y = 10)
+	addtimer(CALLBACK(src, .proc/remove_emote_popup_on_obj, emote_overlay), time)
+
 /mob/living/proc/remove_emote_popup_on_mob(obj/effect/overlay/emote_popup/emote_overlay)
 	vis_contents -= emote_overlay
 	qdel(emote_overlay)
 	return
+
+/obj/proc/remove_emote_popup_on_obj(obj/effect/overlay/emote_popup/emote_overlay)
+	vis_contents -= emote_overlay
+	qdel(emote_overlay)
+	return
```

Comment:
Please document these procs, I can help you with the form of documentation if you want me to!